Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -79,8 +91,76 @@ private void ToggleConsole()
         }
     }
 
-    #region Damage Player
+    #region FreelookCam
+    
+    /// <summary>
+    /// The player unposseses the character
+    /// and become a free look cam that can
+    /// fly arround
+    /// </summary>
+    private void ToggleFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return;
+
+        if (!_isInFreeLookCamMode)
+        {
+             _spawnedFreeLookCam = EnterFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Exit Free Look Cam";
+        }
+        else if (_isInFreeLookCamMode)
+        {
+            ExitFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Enter Free Look Cam";
```

Comment:
If you're going to serialize these objects and they arent necessary for the game, please add a null check to prevent errors before `_freeLookCam` and `_toggleFreeLookCamButton`

Reply:
doesn't use free look cam